weblab/entities/tests/test_views.py

```diff
@@ -1958,3 +1958,36 @@ def test_nonexistent_entity_redirects_anonymous_to_login(self, client, helpers,
     def test_nonexistent_entity_generates_404_for_user(self, client, logged_in_user, helpers, recipe, url):
         response = client.get(url % 10000)
         assert response.status_code == 404
+
+
+@pytest.mark.django_db
+class TestEntityRunExperiment:
```

Comment:
Tests look good!

Reply:
Well Raquel did put in my aims for probation 'increase test coverage on a project by 75%' hard to do with weblab but I'm all for TDD